### PR TITLE
Update docs for steps to take if CI fails

### PR DESCRIPTION
**Pull request by danmoseley**
@jaredpar  @trylek  @safern @ViktorHofer  can you please review and help me get this right?

BTW, is "retrying only failed legs" only possible if you have commit permissions? If so I will mention that.

### CODE REVIEW

File: docs/pr-guide.md [hunk 1]
```diff
@@ -60,15 +60,16 @@ Validation may fail for several reasons
...
 
-* An existing issue in the repository should be searched for. Usually the test method's or the test assembly's name (in case of a crash) are good parameters.
-* If there's an existing issue, a comment should be placed that includes a) the link to the build, b) the affected configuration (ie `netcoreapp-Windows_NT-Release-x64-Windows.81.Amd64.Open`) and c) the Error message and Stack trace. This is necessary as retention policies are in place that recycle _old_ builds. In case the issue is already closed, it should be reopened and labels should be updated to reflect the current failure state. 
-* If there's no existing issue, an issue should be created with the same information outlined above.
-* In a follow-up Pull Request, the failing test(s) should be disabled with the corresponding issue link, e.g. `[ActiveIssue(x)]`, and the tracking issue should be labeled as `disabled-test`.
-* A comment should be placed in the original Pull Request that links to the created or updated issues.
+* Search for an [existing issue](https://github.com/dotnet/runtime/issues). Usually the test method's or the test assembly's name (in case of a crash) are good search parameters.
```
Comment (jaredpar) [inline docs/pr-guide.md]: Can we also link to https://github.com/dotnet/runtime/issues/702

This is our status page for all known CI issues. It's the best place to quickly check to see if a failure is impacting your PR

File: docs/pr-guide.md [hunk 2]
```diff
@@ -25,21 +25,55 @@ Anyone with write access can merge a pull request manually or by setting the [au
...
+### Additional information:
+  * You can list the available pipelines by adding a comment like `/azp list` or get the available commands by adding a comment like `azp help`.
+  * Reach out to the infrastructure team for assistance on [Teams channel](https://teams.microsoft.com/l/channel/19%3ab27b36ecd10a46398da76b02f0411de7%40thread.skype/Infrastructure?groupId=014ca51d-be57-47fa-9628-a15efcc3c376&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) (for corpnet users) or on [Gitter](https://gitter.im/dotnet/community) in other cases.
+
+## What to do if you determine the failure is unrelated
+
+If you have determined the failure is definitely not caused by changes in your PR, please do this:
+
+* Search for an [existing issue](https://github.com/dotnet/runtime/issues). Usually the test method's or the test assembly's name (in case of a crash) are good search parameters.
+  * If there's an existing issue, add a comment with a) the link to the build, b) the affected configuration (ie `netcoreapp-Windows_NT-Release-x64-Windows.81.Amd64.Open`) and c) the Error message and Stack trace. This is necessary as retention policies are in place that recycle _old_ builds.
```
Comment (safern) [inline docs/pr-guide.md]: Sometimes there are crashes, and there is no stack trace, so could you mention including the link to the crashdump and/or console log?

**Review by safern — APPROVED** [review]
Thanks, @danmosemsft

**Comment by danmosemsft** [comment]
If anyone else has feedback, LMK I'll push another update.

**Pull request merged**